deployment/main.py

```diff
@@ -1,5 +1,6 @@
 from flask import Flask
 from learn import build_model, CHECKPOINT_DIR, EMBEDDING_SIZE, RNN_UNITS
```

Comment:
Can we use env variables for `CHECKPOINT_DIR`, `EMBEDDING_SIZE`, `RNN_UNITS` for both training and deployment?